File: c/par_impar.c
#include <stdio.h>

int main(){
    int n, valor;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    for (int i=0; i<n; i++) {
        printf("Digite um numero: ");
        scanf("%d", &valor);

        if (valor == 0) {
            printf("NULO\n");
        }
        else {
            if (valor % 2 == 0) {
                printf("PAR");
            }
            else {
                printf("IMPAR");
            }

            if(valor > 0) {
                printf(" POSITIVO\n");
            }
            else {
                printf(" NEGATIVO\n");
            }
        }
    }

	return 0;
}

File: python/par_impar.py
n: int; valor: int

n = int(input("Quantos numeros voce vai digitar? "))

for i in range(n):
	valor = int(input("Digite um numero: "))

	if valor == 0:
		print("NULO")
	else:
		if valor % 2 == 0:
			print("PAR", end="")
		else:
			print("IMPAR", end="")
		if valor > 0:
			print(" POSITIVO")
		else:
			print(" NEGATIVO")
